datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxRefreshDatahubResponse.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.sdx.api.model;
+
+import static com.sequenceiq.sdx.api.model.ModelDescriptions.DATAHUB_CRNS_REFRESHED;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SdxRefreshDatahubResponse {
+
+    // Crn, Name
+    @ApiModelProperty(DATAHUB_CRNS_REFRESHED)
+    private Map<String, String> refreshedDatahubs;
```

Comment:
I don't really like this approach on the API where it's not obvious what's there.
Would it make sense to create a class, which has crn and name field and have a collection of that here? We might already have such thing, I don't know